package.json

```diff
@@ -46,30 +46,29 @@
     "ember-cli-inject-live-reload": "^1.4.1",
     "ember-cli-mirage": "0.2.5",
     "ember-cli-mocha": "0.13.2",
-    "ember-cli-shims": "^1.0.2",
     "ember-cli-notifications": "^4.1.6",
-    "ember-cli-test-loader": "^1.1.1",
+    "ember-cli-shims": "^1.0.2",
     "ember-cli-uglify": "^1.2.0",
     "ember-code-snippet": "1.7.0",
     "ember-computed-decorators": "~0.3.0",
     "ember-concurrency": "~0.7.15",
-    "ember-data": "^2.11.0",
+    "ember-data": "~2.12.0",
```

Comment:
Is this version compatible with `ember-cli-mirage`?  `ember-cli-mirage` is the package that requires this dev dependency so shouldn't we be using whatever version of ember data version `0.2.5` of `ember-cli-mirage` installs?